crates/pallet-domains/src/lib.rs

```diff
@@ -478,6 +478,12 @@ mod pallet {
     pub(super) type LastEpochStakingDistribution<T: Config> =
         StorageMap<_, Identity, DomainId, ElectionVerificationParams<BalanceOf<T>>, OptionQuery>;
 
+    /// A preferred Operator for a given Nominator. Nominator's rewards are auto staked to their
+    /// preferred Operator.
```

Comment:
Nominator's rewards typically refer to the staking reward in my mind. Since this feels more Farmer-oriented, perhaps something like the following could avoid potential confusion.

```suggestion
    /// A preferred Operator for a given Farmer, enabling automatic staking of block rewards.
    /// For the auto-staking to succeed, the Farmer must also be a Nominator of the preferred Operator.
```